config/README.md

```diff
@@ -1,3 +1,3 @@
 # Where have all the configurations gone?
 
-## https://github.com/MarlinFirmware/Configurations/archive/bugfix-2.0.x.zip
+## https://github.com/MarlinFirmware/Configurations/releases
```

Comment:
https://github.com/MarlinFirmware/Configurations/archive/bugfix-2.0.x.zip is correct for the `bugfix-2.0.x` branch.